extensions.json

```diff
@@ -883,6 +883,17 @@
             "archive": "https://github.com/lnbits/lndhub/archive/refs/tags/v0.4.2.zip",
             "hash": "e1d9d4db9b9c38efbf65645e9b2dc535789295e88090e5a5d49e81996c58444a"
         },
+        {
+            "id": "lndhub",
+            "repo": "https://github.com/lnbits/lndhub",
+            "name": "LndHub",
+            "version": "0.5.0",
+            "min_lnbits_version": "0.12.11",
+            "short_description": "Access lnbits from BlueWallet or Zeus",
```

Comment:
"lnbits" needs to be changed to "LNbits"

Reply:
done